client/src/components/common/ExportFiles.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Menu, MenuButton, MenuList, MenuItem, Button } from '@chakra-ui/react';
+
+const ExportFiles = (props) => {
+  const handleExportCSV = () => {
+    const dateTime = getDateTime();
+    const arrData = props.data;
```

Comment:
Props destructuring that you had is better. Just do the validation. 

Here are some official React.js documentation on that: https://reactjs.org/docs/typechecking-with-proptypes.html

And you can find examples already in the code.

Reply:
Ayt, its fixed